src/components/connect.tsx

```diff
@@ -609,6 +614,10 @@ function connect<
         ? props.store!
         : contextValue!.store
 
+      const getServerSnapshot = didStoreComeFromContext
+        ? contextValue.getServerState
+        : store.getState
```

Comment:
Do you plan to support server snapshots for a `!didStoreComeFromContext` case? I'm not interested in it myself but it would be a surprise to me that, as a user, I couldn't use this functionality when using a `store` prop